src/main/java/com/eternalcode/lobbyheads/reload/ReloadService.java

```diff
@@ -0,0 +1,20 @@
+package com.eternalcode.lobbyheads.reload;
+
+import com.eternalcode.lobbyheads.configuration.ConfigurationService;
+import com.eternalcode.lobbyheads.head.hologram.HologramService;
+
+public class ReloadService {
+
+    private final HologramService hologramService;
+    private final ConfigurationService configurationService;
+
+    public ReloadService(HologramService hologramService, ConfigurationService configurationService) {
+        this.hologramService = hologramService;
+        this.configurationService = configurationService;
+    }
+
+    public void reload() {
+        this.configurationService.reload();
+        this.hologramService.updateHolograms();
+    }
```

Comment:
This method should be moved to HologramService.
And if you want it to stay that way, I suggest changing the name to HologramReloadService.

Reply:
This method does not only reload holograms, it also reloads the config and holograms.

Reply:
So i suggest change name to HologramReloadService.

Reply:
> So I suggest changing the name to HologramReloadService.

Why? this reload config and holograms, not only holograms.

Reply:
I suggest to remove hologram and config dependencies from this service and create interface Reloadable that hologram and config services will implements.